### PR TITLE
Update ComputerVision C# SDK auto-generated code.

### PR DESCRIPTION
**Pull request by yangyuan**
REST spec PR https://github.com/Azure/azure-rest-api-specs/pull/3440

Please hold for @cthrash's review.

## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
I don't see any version change. I also see breaking change.
Please update version, release notes.

**Comment by yangyuan** [comment]
Hi @shahabhijeet, I explained in the description, we are not gonna publish SDK after this PR.
We will wait for another change and publish together. Can we do that?

**Comment by yangyuan** [comment]
Hi @shahabhijeet, I've bumped the versions...
Please noticed that 3.0.0 are unlisted, so the release notes contains the changes from 2.0.0.

File: src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Cognitive Services ComputerVision SDK")]
 [assembly: AssemblyDescription("Provides access to the Microsoft Cognitive Services ComputerVision APIs.")]
 
-[assembly: AssemblyVersion("3.0.0.0")]
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision/Properties/AssemblyInfo.cs]: AssemblyVersion remains the same but AssemblyFileVersion must be bumped

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please update the branch and address the comments

**Comment by yangyuan** [comment]
Hi @dsajanice, I fixed the issue, thanks.

**Comment by dsgouda** [comment]
@yangyuan Please pull from upstream psSdkJson6 and update this PR

**Comment by yangyuan** [comment]
Hi @dsgouda, I did pull it from psSdkJson6, but anyway I have debased the branch to newest psSdkJson6.

**Review by dsgouda — APPROVED** [review]

**Comment by dsgouda** [comment]
@shahabhijeet please approve

**Comment by yangyuan** [comment]
ping @shahabhijeet :)

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**